qcp/main.py

```diff
@@ -1,4 +1,18 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
+
+# # Copyright 2022 Tiernan8r
```

Comment:
License additions spun out to https://github.com/Tiernan8r/quantum_computing_project/pull/22